### DISCUSSION TITLE
When editing workflow, browser cache is overwriting parameter values.

### DISCUSSION THREAD

**Discussion by david-grokstream** [StackStorm/st2#6256] [UI]
We are using 3.8 on ubuntu, it is in a docker installation but I am not sure that matters for this problem.

We have orchestra workflows listed in the action tab, and we can open the editor for them and the workflow editor opens in a new tab.

In the workflow editor, we are able to edit and change default parameter values and save the workflow with no problems.  Checking files on disk, the data is correctly stored.   Refreshing the UI on the action list also shows the changes.

However, if we close the workflow editor tab and reopen it, the parameter values from the original load are populated in the parameters tab and will overwrite the previous values.

If we open in an incognito tab, the data load properly with the new parameter values.

Is this a known issue?  We can do hard refreshes, etc, but it was forgotten a few times and messed up some actions.

We are all using Chrome as the browser on both windows and mac, and I tested Edge and Brave on windows, and we experience the same problem.  A search has not revealed any knowledge on how to fix this.

Any ideas that do NOT include disabling local cache on all browsers?

Dave

**Comment by fdrab** [comment]
relevant PR probably https://github.com/StackStorm/st2/pull/6264
